Content.Server/Shipyard/Systems/ShipyardSystem.Consoles.cs

```diff
@@ -39,6 +39,7 @@
 using Content.Shared.UserInterface;
 using Robust.Shared.Audio;
 using Robust.Shared.Audio.Systems;
+using Microsoft.CodeAnalysis.Diagnostics;
```

Comment:
What the hell, where did this come from? Ugh. I can make a new PR to remove this stupid extra import.